webcompat/templates/layout.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>webcompat.com</title>
+  <title>{% block title %}{% endblock %}webcompat.com</title>
```

Comment:
Ah sure, I considered that but thought the homepage was nice as just 'webcompat.com' - for the index page then shall I add something like 'Home'?